etc/kayobe/environments/aufn-ceph/controllers.yml

```diff
@@ -0,0 +1,2 @@
+---
+controllers_bootstrap_user: "{{ lookup('env', 'USER') }}"
```

Comment:
Is the same not required for storage and seed hosts?

Reply:
I suppose it should be, although I didn't hit any issues without them defined...

Reply:
It does make me wonder, why would control/compute be different than storage?

Reply:
Just running through this again now, and it looks like we don't want to change the user for the seed. Fails in `seed host configure`:

Reply:
```TASK [Check if python is installed] *************************************************************************************************************************************************************************************
Wednesday 03 May 2023  12:57:44 +0000 (0:00:00.115)       0:00:02.268 *********
fatal: [seed]: UNREACHABLE! => changed=false 
  msg: 'Failed to connect to the host via ssh: cloud-user@192.168.33.5: Permission denied (publickey,gssapi-keyex,gssapi-with-mic).'                                                                                    
  unreachable: true```

Reply:
I've ran through an AUFN again to nail down what's up with the different bootstrap users. Turns out I'd managed to misspell `controllers_bootstrap_user` (should be `controller` singular) so it was using `rocky` rather than `cloud-user`. I suspect I'd made a similar mistake with the computes the first time around...

The correct user for the latest Rocky9 image is `rocky`, so I've updated accordingly.